.github/workflows/release.yaml

```diff
@@ -43,4 +62,66 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: build/release.sh $GH_TAG
+  
+  # # # # # # # # # # # # # # # # #
+  # Update Helm Chart
+  # # # # # # # # # # # # # # # # #
+  update-helm-chart-version:
```

Comment:
Why is this done twice?

Reply:
Copypasta error by me. I've removed the 2nd instance.